model/quic-socket-base.cc

```diff
@@ -296,6 +308,16 @@ QuicSocketBase::GetTypeId (void)
                      "Receive QUIC packet from UDP protocol",
                      MakeTraceSourceAccessor (&QuicSocketBase::m_rxTrace),
                      "ns3::QuicSocketBase::QuicTxRxTracedCallback")
+    .AddAttribute ("CongControl",
+                   "The Object managing CC operations",
+                   PointerValue (),
```

Comment:
Are you sure a default is not needed?

Reply:
The attribute was actually unused, it was probably a residue from some tests. Removed